src/object.rs

```diff
@@ -328,60 +328,59 @@ pub struct ObjectHandle {
-        m.insert((5, 0x04), ObjectCapability::DeleteWrapKey);
-        m.insert((5, 0x08), ObjectCapability::DeleteHmacKey);
-        m.insert((5, 0x10), ObjectCapability::DeleteTemplate);
-        m.insert((5, 0x20), ObjectCapability::DeleteOtpAeadKey);
-        m
+    static ref CAPABILITIES_MAP: HashMap<(u8, u8), ObjectCapability> =
+        [((0, 0x01), ObjectCapability::GetOpaque),
+         ((0, 0x02), ObjectCapability::PutOpaque),
+         ((0, 0x04), ObjectCapability::PutAuthenticationKey),
+         ((0, 0x08), ObjectCapability::PutAsymmetricKey),
+         ((0, 0x10), ObjectCapability::GenerateAsymmetricKey),
+         ((0, 0x20), ObjectCapability::SignPkcs),
+         ((0, 0x40), ObjectCapability::SignPss),
+         ((0, 0x80), ObjectCapability::SignEcdsa),
+
+         ((1, 0x01), ObjectCapability::SignEddsa),
+         ((1, 0x02), ObjectCapability::DecryptPkcs),
+         ((1, 0x04), ObjectCapability::DecryptOaep),
+         ((1, 0x08), ObjectCapability::DeriveEcdh),
+         ((1, 0x10), ObjectCapability::ExportWrapped),
+         ((1, 0x20), ObjectCapability::ImportWrapped),
+         ((1, 0x40), ObjectCapability::PutWrapKey),
+         ((1, 0x80), ObjectCapability::GenerateWrapKey),
+
+         ((2, 0x01), ObjectCapability::ExportableUnderWrap),
+         ((2, 0x02), ObjectCapability::SetOption),
+         ((2, 0x04), ObjectCapability::GetOption),
+         ((2, 0x08), ObjectCapability::GetPseudoRandom),
+         ((2, 0x10), ObjectCapability::PutHmacKey),
+         ((2, 0x20), ObjectCapability::GenerateHmacKey),
+         ((2, 0x40), ObjectCapability::SignHmac),
+         ((2, 0x80), ObjectCapability::VerifyHmac),
+
+         ((3, 0x01), ObjectCapability::GetLogEntries),
+         ((3, 0x02), ObjectCapability::SignSshCertificate),
+         ((3, 0x04), ObjectCapability::GetTemplate),
+         ((3, 0x08), ObjectCapability::PutTemplate),
+         ((3, 0x10), ObjectCapability::ResetDevice),
+         ((3, 0x20), ObjectCapability::DecryptOtp),
+         ((3, 0x40), ObjectCapability::CreateOtpAead),
+         ((3, 0x80), ObjectCapability::RandomizeOtpAead),
+
+         ((4, 0x01), ObjectCapability::RewrapFromOtpAeadKey),
+         ((4, 0x02), ObjectCapability::RewrapToOtpAeadKey),
+         ((4, 0x04), ObjectCapability::SignAttestationCertificate),
+         ((4, 0x08), ObjectCapability::PutOtpAeadKey),
+         ((4, 0x10), ObjectCapability::GenerateOtpAeadKey),
+         ((4, 0x20), ObjectCapability::WrapData),
+         ((4, 0x40), ObjectCapability::UnwrapData),
+         ((4, 0x80), ObjectCapability::DeleteOpaque),
+
+         ((5, 0x01), ObjectCapability::DeleteAuthenticationKey),
+         ((5, 0x02), ObjectCapability::DeleteAsymmetricKey),
+         ((5, 0x04), ObjectCapability::DeleteWrapKey),
+         ((5, 0x08), ObjectCapability::DeleteHmacKey),
+         ((5, 0x10), ObjectCapability::DeleteTemplate),
+         ((5, 0x20), ObjectCapability::DeleteOtpAeadKey)]
+         .iter().cloned().collect();
     };
```

Comment:
This doesn't build, there is an unmatched brace.

Can you please update and make sure it builds?